src/Connection/HiveDsnFactory.php

```diff
@@ -27,8 +27,9 @@ public function create(
         $parameters['Port'] = $dbConfig->getPort();
         $parameters['Schema'] = $dbConfig->getDatabase();
         $parameters['UseNativeQuery'] = '1';
-        $parameters['DefaultStringColumnLength'] = '65536';
-        $parameters['BinaryColumnLength'] = '65536';
+        $parameters['DefaultStringColumnLength'] = '16777216';
+        $parameters['DefaultVarcharColumnLength'] = '16777216';
+        $parameters['BinaryColumnLength'] = '16777216';
```

Comment:
Timhle si nejsem jistej, jestli to ma opravdu vlyv, ale funguje to takhle, uz to tak necham. 
Kdyby se objevil problem, tak to muzem postetlovat